content/en/real_user_monitoring/session_replay/_index.md

```diff
@@ -56,7 +56,7 @@ datadogRum.startSessionReplayRecording();
 
 Replace the Browser SDK URL `https://www.datadoghq-browser-agent.com/datadog-rum.js` with `https://www.datadoghq-browser-agent.com/datadog-rum-v4.js`. When `DD_RUM.init()` is called, the Session Replay recording does not start until `DD_RUM.startSessionReplayRecording()` is also called.
 
-The Session Replay recorder supports all browsers supported by the RUM Browser SDK with the exception of IE11. For more information, see the [browser support table][3].
+The Session Replay recorder supports all browsers supported by the RUM Browser SDK. For more information, see the [browser support table][3].
```

Comment:
we shouldn't change anything on the RUM side.
This is a synthetics thing only :)

Reply:
otherwise we are good to go

Reply:
```suggestion
The Session Replay recorder supports all browsers supported by the RUM Browser SDK with the exception of IE11. For more information, see the [browser support table][3].
```